WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -7,6 +7,15 @@ class ReaderTabViewController: UIViewController {
     init(viewModel: ReaderTabViewModel) {
         self.viewModel = viewModel
         super.init(nibName: nil, bundle: nil)
+        self.viewModel.filterTapped = { [weak self] completion in
```

Comment:
can we possibly move the creation of specific instances of `FilterSheetViewController` and `BottomSheetViewController` out of `ReaderTabViewController`?
It would be preferable for two reasons:
- it would be more consistent with the 'MVVM-Like' architecture we are using
- it would leave `ReaderTabViewController` unaware of any model and child viewController. If we confine these in `ReaderTabViewModel` it maybe easier to evolve.

if we feel that `ReaderTabViewModel` grows too much, we can also break it up

Here is a possible implementation; does not have to be like this, it's just an idea

[bottomSheet.patch.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/4463966/bottomSheet.patch.zip)